app/README.md

```diff
@@ -56,8 +56,6 @@ Note: make sure TypeScript and Javascript Language Features are enabled in VS Co
   - Disallows usage of `any` type. The usage of `any` defeats the purpose of typescript. Consider using `unknown` type instead instead.
 - "react/resct-in-jsx-scope": "off"
   - suppress errors for missing 'import React' in files because NextJS does this for us.
-- "space-before-function-paren": ["error", "never"]
-  - suppresses errors for lack of space before function parenthesis to allow for `function()`
```

Comment:
Why was this removed? Is it no longer needed?

Reply:
I went with the opposite approach:

> disabling style lint rules and using whatever the code formatter's default is

See [discussion](https://github.com/navapbc/template-application-nextjs/pull/24#discussion_r920556414).